### DISCUSSION TITLE
Retrieve domains associated with deleted team

### DISCUSSION THREAD

**Discussion by Seth-McKilla** [vercel/community#3389] [Help] [answered]
### Summary

Hello,

I recently downgraded a team app back to my free hobby account since the project wasn't getting much traction. In order to cancel the pro plan I was told that I needed to delete the team. However, I had two domains (through Vercel) that were associated with the team which was are used to access my app and linked to Google Suite for emailing. I didn't transfer these domains to my personal hobby account prior to deleting the team so now the DNS configuration is invalid as these domains cannot be found.

Is there a way to retrieve these domains / transfer them to my personal account to relink to the app or are they lost forever? I think it would also be a good feature to alert the user that they have domains associated with the team and ask if they would like to transfer them before confirming the deletion. There may have been a message saying this and I just missed it 😓

Thanks,
Seth

### Steps to Reproduce

1. Create team
2. Create and deploy app associated with team
3. Purchase domain through team
4. Assign domain to deployed app in step 2
5. Transfer app to personal account
6. Delete team

**Comment by amyegan** [comment]
Hey @Seth-McKilla. If you let me know the domains, I can look into this for you

**Reply by Seth-McKilla** [reply]
Thanks for the response @amyegan! The two domains are realtyflow.co and realtyflow.app

**Reply by amyegan** [reply, marked as answer]
I found the original project and just sent you ownership transfer requests for these domains. They will be associated with your personal account after you accept the transfer. Please let me know if there is any other help you need 🙂

**Reply by Seth-McKilla** [reply]
You're a lifesaver, thanks so much!

**Comment by Joe-at-RAR** [comment]
> I found the original project and just sent you ownership transfer requests for these domains. They will be associated with your personal account after you accept the transfer. Please let me know if there is any other help you need 🙂

I additionally need help with this! The domains are silknote.com.au an silknotes.com.au - I deleted the project they were transferred from.

**Reply by amyegan** [reply]
Hi @Joe-at-RAR. It looks like those domains belong to a team that is still active. If you're part of that team, you can transfer them yourself. If you left the team, please contact the current owner for help transferring the domain.
